services/repository/src/test/java/org/sagebionetworks/repo/web/filter/throttle/RequestThrottleFilterTest.java

```diff
@@ -59,13 +64,17 @@ public void setUp() throws Exception{
 		mockRequest.setRemoteAddr(ipAddress);
 		mockRequest.setRequestURI(path);
 		mockRequest.setCookies(new Cookie(SESSION_ID_COOKIE_NAME, sessionId));
+		mockRequest.addHeader(AuthorizationConstants.SYNAPSE_AUTHORIZATION_HEADER_NAME, "Bearer "+ACCESS_TOKEN);
 
 		ReflectionTestUtils.setField(filter, "consumer", mockConsumer);
+		ReflectionTestUtils.setField(filter, "oidcManager", mockOidcManager);

```

Comment:
We can use @InjectMocks and get rid of the ReflectionTestUtils.setField

Reply:
Done.